src/rhsm/logutil.py

```diff
@@ -197,6 +199,10 @@ def init_logger(config=None):
     for error_message in pending_error_messages:
         log.error(error_message)
 
+    #read_log_level = config.get('logging', 'default_log_level')
```

Comment:
We should remove this comment, it seems like leftover code which was moved above.

Reply:
yes removed